src/client/components/ImageMenu.js

```diff
@@ -102,29 +106,53 @@ const ImageMenu = ({
     imageExists &&
     boxes[ethereumAddress].publicProfile[imageTag][0].contentUrl['/']
 
+  const handleToggleMenu = imageTag => {
+    imageTag === activeImageMenu
+      ? toggleImageMenu('')
+      : toggleImageMenu(imageTag)
+  }
+
   return (
-    <ImageMenuStyled
-      top={top}
-      right={right}
-      dragging={dragState.dragging}
-      {...getRootProps({
-        className: 'dropzone',
-        isDragActive,
-        isDragAccept,
-        isDragReject,
-        imageCid,
-      })}
-    >
-      {' '}
-      <div>Update {imageTitle} photo</div>
-      <input {...getInputProps({ disabled: false })} />
-      <div onClick={open}>Upload new image</div>
-      {imageExists && (
-        <div onClick={() => dispatchModal(removeItem(imageCid, imageTag))}>
-          Delete
+    <ClickOutHandler onClickOut={() => toggleImageMenu('')}>
```

Comment:
I agree that it seems to make sense to add a dependency for this functionality, but how did you choose this dependency specifically? Are you sure it's sustainable? I get concerned that the last time an update was made to this package was december 2018, and its react version is pre-hooks (which usually is a danger sign).

Reply:
No dependency necessary! Add a wrapping div with a `tabIndex="-1"`. This will make it respond to `onFocus` and, more importantly`, `onBlur` events. 

```jsx
<div tabIndex="-1" onBlur={() => toggleImageMenu('')}>
  <ImageMenuStyled ... />
</div>
```

Reply:
A-ha. I took ClickOutHandler from Aragon's DropDown, and now replaced it as suggested - and it working - thank you! :)